andela/ah-orcas-frontend
# #161254622 create articles

#### What does this PR do?
Allow user to view and create articles
#### Description of Task to be completed?
```Create article```
```View articles```
```view single article```
```delete article```
```update article```
#### How should this be manually tested?
- Clone this repository
`git clone https://github.com/andela/ah-orcas-frontend.git`
- Change the working directory to the project folder
`cd ah-orcas-frontend`
- Checkout to the appropriate branch
`git checkout ft-create-articles-161254622`
- Install the required dependencies:
     `npm start`
should be able to see the first ten articles in home
if authenticated you should be able to create
 
#### What are the relevant pivotal tracker stories?
 [#161254616](https://www.pivotaltracker.com/story/show/161254616)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30722290/49795137-444b6880-fd4a-11e8-86a5-6f5b75ceefa7.png)
![image](https://user-images.githubusercontent.com/30722290/49795158-53cab180-fd4a-11e8-93a4-233abe465e5f.png)
![image](https://user-images.githubusercontent.com/30722290/50051202-f86c3b00-011e-11e9-85ec-6756813572e3.png)
![image](https://user-images.githubusercontent.com/30722290/50051313-aaf0cd80-0120-11e9-9260-09f07e66edd6.png)

#### Checklist:

- [] I have made corresponding changes to the documentation
